frontend/src/app/modules/online-editor/online-editor-page/online-editor-page.component.ts

```diff
@@ -52,6 +59,16 @@ export class OnlineEditorPageComponent implements OnDestroy, OnInit {
     }
 
     ngOnInit() {
+        this.signalRService.start();
+
+        this.signalRService.listenMessages((message: string) => {
```

Comment:
cannot you just use the interface here?

Reply:
I thought SignalR sends messages as strings, so I will still need to parse messages